lib/liquid/standardfilters.rb

```diff
@@ -341,9 +341,13 @@ def floor(input)
       raise Liquid::FloatDomainError, e.message
     end
 
-    def default(input, default_value = "".freeze)
-      is_blank = input.respond_to?(:empty?) ? input.empty? : !input
-      is_blank ? default_value : input
+    def default(input, default_value = EMPTY_STR)
```

Comment:
Whoops, good catch.

`''.freeze` is used several times throughout this file; it may not be a bad idea to have an `EMPTY_STR` constant, but that is a discussion for a different PR.